src/App.js

```diff
@@ -1,16 +1,25 @@
+import { Routes, Route } from 'react-router-dom';
+
 import styles from './styles/App/App.module.css';
+import Home from './components/Home';
 import Calculator from './components/Calculator';
 import Quote from './components/Quote';
+import Navbar from './components/Navbar';
 
 function App() {
   return (
-    <main className={styles.app}>
-      <h1 className={styles.pageHeader}>Math Magicians</h1>
-      <div className={styles.content}>
-        <Calculator />
-        <Quote />
-      </div>
-    </main>
+    <>
+      <Navbar />
+      <main className={styles.app}>
+        <div className={styles.content}>
+          <Routes>
+            <Route path="/" element={<Home />} />
+            <Route path="calculator" element={<Calculator />} />
+            <Route path="quote" element={<Quote />} />
+          </Routes>
+        </div>
+      </main>
+    </>
```

Comment:
Hi @tresorsawasawa, I forgot to provide you with my **.env** file. I will include that and ask for a second review.

Reply:
- [ ] As pointed out by the previous reviewer, kindly ensure all your routes open their expected pages. Currently, when a user navigates to the  QuotePage it throws an error. Consider fixing this error as it will help create a good user experience. Below is the error that gets thrown to the screen.
<img width="1440" alt="Screenshot 2023-07-18 at 22 13 41" src="https://github.com/henokkhm/microverse-math-magicians/assets/107192076/15bf6150-ec9d-4a41-a73e-2c485b3c2686">